x-pack/docs/en/rest-api/security/get-api-keys.asciidoc

```diff
@@ -191,7 +191,7 @@ A successful call returns a JSON structure that contains the information of one
       }
     },
     {
-      "id": "api-key-id-2",
+      "id": "6wHJmcQpReKBa42EHV5SBw",
```

Comment:
This change is not really related to the purpose of this PR. It's more of a bug fix of existing doc because this ID format is misleading.